### PR TITLE
Added support for RequireJS, CommonJS

### PR DESCRIPTION
**Pull request by byonchev**

### CODE REVIEW

**Comment by moust** [comment]
:+1:

File: src/intro.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(function (factory) {
+   if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['moment'], factory);
+    } else if ( typeof exports === 'object' ) {
+        // Node/CommonJS
+        module.exports = factory();
```
Comment (ChiperSoft) [inline src/intro.js]: Shouldn't you be doing `factory(require('moment'))` here?  The moment-less build wont have moment bundled in, so moment will be undefined.

Comment (byonchev) [inline src/intro.js]: Oops, I was focused on the AMD part and I missed that.

**Comment by ChiperSoft** [comment]
Thank you for making this. I had been wanting to add a UMD wrapper but hadn't gotten around to it.  I think there's still some ways this could be cleaned up more, but this works for now.

**Comment by ChiperSoft** [comment]
This is now released in version 0.6

**Pull request merged**